[2025-01-17T18:13:32.274+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-17T18:13:32.290+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_ELT_PostgreSQL.get_api_python manual__2025-01-17T18:13:29.725274+00:00 [queued]>
[2025-01-17T18:13:32.298+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_ELT_PostgreSQL.get_api_python manual__2025-01-17T18:13:29.725274+00:00 [queued]>
[2025-01-17T18:13:32.299+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-17T18:13:32.311+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_api_python> on 2025-01-17 18:13:29.725274+00:00
[2025-01-17T18:13:32.319+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3194) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-17T18:13:32.320+0000] {standard_task_runner.py:72} INFO - Started process 3199 to run task
[2025-01-17T18:13:32.321+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'DAG_ELT_PostgreSQL', 'get_api_python', 'manual__2025-01-17T18:13:29.725274+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/ETL_PostgreSQL.py', '--cfg-path', '/tmp/tmpp93_boq6']
[2025-01-17T18:13:32.322+0000] {standard_task_runner.py:105} INFO - Job 32: Subtask get_api_python
[2025-01-17T18:13:32.360+0000] {task_command.py:467} INFO - Running <TaskInstance: DAG_ELT_PostgreSQL.get_api_python manual__2025-01-17T18:13:29.725274+00:00 [running]> on host 59ff16f1d4fb
[2025-01-17T18:13:32.711+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Wilfrido Rondon' AIRFLOW_CTX_DAG_ID='DAG_ELT_PostgreSQL' AIRFLOW_CTX_TASK_ID='get_api_python' AIRFLOW_CTX_EXECUTION_DATE='2025-01-17T18:13:29.725274+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-17T18:13:29.725274+00:00'
[2025-01-17T18:13:32.712+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-17T18:13:32.712+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-17T18:13:32.713+0000] {logging_mixin.py:190} INFO - Current task name:get_api_python state:running start_date:2025-01-17 18:13:32.290944+00:00
[2025-01-17T18:13:32.713+0000] {logging_mixin.py:190} INFO - Dag name:DAG_ELT_PostgreSQL and current dag run status:running
[2025-01-17T18:13:32.713+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-17T18:13:33.458+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-17T18:13:33.468+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-17T18:13:33.468+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=DAG_ELT_PostgreSQL, task_id=get_api_python, run_id=manual__2025-01-17T18:13:29.725274+00:00, execution_date=20250117T181329, start_date=20250117T181332, end_date=20250117T181333
[2025-01-17T18:13:33.480+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-17T18:13:33.480+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-17T18:13:33.481+0000] {logging_mixin.py:190} INFO - Dag name:DAG_ELT_PostgreSQL queued_at:2025-01-17 18:13:29.737868+00:00
[2025-01-17T18:13:33.481+0000] {logging_mixin.py:190} INFO - Task hostname:59ff16f1d4fb operator:PythonOperator
[2025-01-17T18:13:33.525+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-17T18:13:33.553+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-17T18:13:33.558+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
